staging/src/k8s.io/apiserver/pkg/storage/watch_cache_test.go

```diff
@@ -51,7 +67,11 @@ func newTestWatchCache(capacity int) *watchCache {
 		return NamespaceKeyFunc("prefix", obj)
 	}
 	getAttrsFunc := func(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
-		return nil, nil, false, nil
+		pod, ok := obj.(*v1.Pod)
+		if !ok {
+			return nil, nil, false, fmt.Errorf("not a pod!")
+		}
+		return labels.Set(pod.Labels), fields.Set{"nodeName": pod.Spec.NodeName}, false, nil
```

Comment:
nit, either make this `"spec.nodeName"` to match reality, or make up a completely bogus field selector like "nodeNameForTest" to make it clear this is not a real pod field set

Reply:
done